app/views/notes/_coauthored_notes.html.erb

```diff
@@ -47,7 +47,7 @@
               <p><%= raw strip_tags(sanitize(RDiscount.new(node.body_preview).to_html)) if node.body %></p>
             <% end %>
             <p>
-              <% if logged_in_as(['admin','moderator']) %><a class="btn btn-default btn-xs" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %>
+              <% if logged_in_as(['admin','moderator']) %><a class="btn btn-default" href="/moderate/spam/<%= node.id %>"><i class="fa fa-ban"></i> <%= t('notes._notes.spam') %></a><% end %>
```

Comment:
hi @trqkkr, its good so far but I doubt that if both buttons are in same line after changing style. Have you installed plots2 on your machine? If yes, please confirm by running otherwise no issue, I will let you know if any other change required. Thanks.

Reply:
@trqkkr while at it, you could post a screenshot of the page here.

Reply:
I don't have ruby on my computer, but this is what the changed code looks like when opening it through chrome:

![screenshot 7](https://user-images.githubusercontent.com/16644581/39007689-4eb83b16-43bb-11e8-8e26-9ca12e5c7b41.png)

Reply:
yes, if you want to see the website running then you have to install rails, ruby and all other dependencies first and then have to follow our plots2 installation instructions to finally run on localhost.